grpcdogfood/server/chart/templates/deployment.yaml

```diff
@@ -0,0 +1,81 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
```

Comment:
These were originally separate files, but I thought these items are very coupled so I threw them into the `deployment.yaml` file. Happy to break it out if you think that's more stylistically correct.